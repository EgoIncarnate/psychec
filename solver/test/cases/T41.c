void f() {
  a.x = 9.9;
  b.y = a;
}
