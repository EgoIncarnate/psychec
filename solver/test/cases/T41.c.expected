struct __T0__ {double x;} a ;
struct __T1__ {struct __T0__ y;} b ;
